### ISSUE TITLE
Get points actual positions in playAnimationStep()

### ISSUE THREAD

**Issue by varjak** [ppconde/fold#175]
Function `playAnimationStep()` is inside origami.ts. 

Currently, `playAnimationStep()` gets the points **original** positions, not the updated positions from each animation step.

One way would be to search for them at each animation step. Another would be to save them to an object (by reference?) at the end of the constructor, so that at each animation step, it would only be necessary to index them from this object.

**Comment by ppconde** [comment]
Hey Guta, just an update on this one. I'm migrating our project to react three fiber so I think I'll try look onto this issue while migrating, is that ok?

**Comment by varjak** [comment]
Hey Conde, yeah of course. When we finish it I'll be able to debug the solver: it is taking very long right now.

**Comment by dedukun** [comment]
Yeah, I'm taking a while to finish this, I have some stuff in a branch in my fork that might be good for merging, I'll have a look after work.
